common/connectors-config/src/main/java/com/vmware/connectors/common/web/ConnectorRootController.java

```diff
@@ -31,10 +31,16 @@
 public class ConnectorRootController {
 
     private final boolean hasTestAuth;
+    private final long maxAge;
+    private final TimeUnit unit;
 
     @Autowired
-    public ConnectorRootController(@Value("${connector.hasTestAuth:false}") boolean hasTestAuth) {
+    public ConnectorRootController(@Value("${connector.hasTestAuth:false}") boolean hasTestAuth,
+                                   @Value("${rootDiscovery.cacheControl.maxAge:1}") long maxAge,
```

Comment:
Its high time that we can start documenting the configuration parameters we are using in our repo.